.circleci/config.yml

```diff
@@ -48,9 +47,39 @@ jobs:
           command: docker run --rm --tty --interactive application:$CIRCLE_SHA1 python --version | grep "3.4"
       - run:
           name: Tag and push Docker images
-          command: .circleci/tag_and_push_docker_image application:$CIRCLE_SHA1
+          command: |
+            .circleci/tag_and_push_docker_image application:$CIRCLE_SHA1 $DSD_DOCKER_REGISTRY/$DSD_DOCKER_IMAGE
+            .circleci/tag_and_push_docker_image application:$CIRCLE_SHA1 $ECR_DOCKER_REPO
+  build_nginx:
+    docker:
+      - image: docker:18.05.0-git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 18.05.0-ce
+          docker_layer_caching: true
+      - run:
+          name: Login to the ECR Docker registry
+          command: |
+            apk add --no-cache --no-progress py2-pip
+            pip install awscli
+            ecr_login="$(AWS_ACCESS_KEY_ID=$NGINX_AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY=$NGINX_AWS_SECRET_ACCESS_KEY aws ecr get-login --region eu-west-1 --no-include-email)"
+            ${ecr_login}
+      - run:
+          name: Build Docker image
+          command: |
+            docker build --file Dockerfile.nginx --tag nginx:$CIRCLE_SHA1 \
+              --label build.git.sha=$CIRCLE_SHA1 \
+              --label build.git.branch=$CIRCLE_BRANCH \
+              --label build.url=$CIRCLE_BUILD_URL \
+              .
+      - run:
+          name: Tag and push Docker images
+          command: .circleci/tag_and_push_docker_image \
+              nginx:$CIRCLE_SHA1 \
+              926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-get-access/laa-fala-nginx
```

Comment:
Sorry, I meant defining `ECR_DOCKER_REPO_BASE=926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-get-access/laa-fala` environment and using `${ECR_DOCKER_REPO_BASE}-nginx` here.

For consistency, could we either use `ECR_DOCKER_REPO_BASE` everywhere or use the full inline image everywhere (line 52), please?

Reply:
Ok. Will add env var into via Circle CI dashboard for `ECR_DOCKER_REPO_BASE` and use throughout.